scripts/gulpfiles/build_tasks.js

```diff
@@ -172,7 +172,7 @@ function stripApacheLicense() {
  */
 var JSCOMP_ERROR = [
   // 'accessControls',  // Deprecated; means same as visibility.
-  'checkPrototypalTypes',
+  // 'checkPrototypalTypes',
```

Comment:
In general our goal should be for Closure Compiler not to routinely emit any diagnostics when run as `npm run build-debug` (same settings as used in `npm test`), so if these diagnostics (`checkPrototypalTypes`, `globalThis` and `missingProperties`) are now being generated we should probably be actively suppressing them by adding them to `JSCOMP_OFF`.

In any case it would be useful to have a comment, either here or in `JSCOMP_OFF` explaining why they have been disabled and indicating when (if ever) we anticipate being able to reenable them.

Reply:
> indicating when (if ever) we anticipate being able to reenable them.

Personally, I want to get rid of closure and not worry about reenabling them. All of these checks are covered by the typescript compiler. Closure only complains because the typescript compiler doesn't transpile thing in a way it can understand.

I will add comments though for context.